portfolio.html

```diff
@@ -0,0 +1,231 @@
+          <h2>Uber Navigation</h2>
+          <div class="frame2">
+            <span>Uber</span>
+            <ul class="role">
+              <li>Lead Developer</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+           Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+          </p>
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button type="button"><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="grid-container2">
+      <div class="about-me">
+        <h1>
+          About Myself
+        </h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <a href="#" class="connect">LET’S CONNECT</a>
+        <div class="social-media2">
+          <ul>
+            <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/Linkedin%20icon.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/medium.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/angellist.svg" alt=""></a></li>
+          </ul>
+        </div>
+          <button type="button" class="btn"><a href="#">Get my resume</a></button>
+        </div>
+      </div>
+      <div class="skills">
+        <ul class="language">
+          <li>
+            <a class="languages" href="#">Languages<img class="arrow" src="images/Icon%20-%20Chevron%20-%20Down%20-%20Larger.svg" alt=""></a>
+            <ul class="sub-lang">
+              <li><a href=""><img src="images/Javascript.svg" class="animation" alt=""><span>JavaSript</span></a></li>
+              <li><a href="#"><img src="images/Html.svg" class="animation" alt=""><span>HTML</span></a></li>
+              <li><a href="#"><img src="images/CSS.svg" class="animation" alt=""/><span>CSS</span></a></li>
+            </ul>
+          </li>
+          <li><a class="frame" href="#">Frameworks <img class="arrow" src="images/frameworks_arrow.svg"></a></li>
+          <hr class="line2">
+          <li><a class="skl" href="#">Skills <img class="arrow2" src="images/frameworks_arrow.svg"></a></li>
+          <hr class="line2">
+        </ul>
+      </div>
+    </section>
+    <section class="footer ">
+      <div class="footer-container">
+        <h1>Contact me</h1>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+       
```

Comment:
- [ ] Please, make sure the about section match the figma design.  Please update the with and font-family of the paragraphe.
**Your design**
![image](https://user-images.githubusercontent.com/74411135/229212837-5768b8cf-978e-491b-9fca-5c9005ce0464.png)

**Figma design**
![image](https://user-images.githubusercontent.com/74411135/229212671-1f3bd50d-6330-41fd-8eba-82eb37ee7e13.png)

This will make your design prettier which will improve the user experience 📱 💻 🖥️ 😃 😺